integrationTests/singleShard/gasSchedule.toml

```diff
@@ -0,0 +1,592 @@
+[BuiltInCost]
```

Comment:
no need for this to be copied here.

Reply:
you have it under integrationTests/vm/arwen

Reply:
Added global variable `arwen.GasSchedulePath` to allow that.